appveyor.yml

```diff
@@ -19,29 +19,30 @@ environment:
     - TOOLCHAIN: "nmake-vs-12-2013"
       PROJECT_DIR: examples\benchmark
 
-    - TOOLCHAIN: "vs-10-2010"
-      PROJECT_DIR: examples\benchmark
+    # - TOOLCHAIN: "vs-10-2010"
```

Comment:
It would be nice to leave the link to the broken job so if somebody want to fix it he can easily find the logs.

Example:
* https://github.com/ingenue/hunter/blob/31dcda75318c2ae1095a67a47d64ab9f29409542/.travis.yml#L92-L95